requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 git+https://github.com/asdf-format/asdf
-git+https://github.com/astropy/astropy
 git+https://github.com/spacetelescope/roman_datamodels
+astropy>=0.0.dev0
+git+https://github.com/spacetelescope/stpipe
```

Comment:
We do have a real astropy dependency and I understand making it part of dev requirements.  I don't think we have any stpipe dependency in the simulator and think we could leave that line out.

Reply:
good catch, fixed